src/components/donate/onlyFiat.js

```diff
@@ -171,12 +171,18 @@ const OnlyFiat = props => {
                 setAmountTyped('')
                 setAmountSelect(i)
               }}
-              sx={{
+              style={{
                 backgroundColor: isSelected ? 'white' : 'transparent',
-                color: isSelected ? 'secondary' : 'white'
+                color: isSelected ? theme.colors.secondary : 'white'
```

Comment:
I know its tempting but please refrain from hardcoding color values. White is represented in the colors if you look at gatsby-plugin-themeui/index.js you will find that you can use the background color instead.

Reply:
This one slipped, my bad 😄